app/assets/javascripts/discourse/app/raw-views/plugin-outlet.gjs

```diff
@@ -0,0 +1,18 @@
+import EmberObject from "@ember/object";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import rawRenderGlimmer from "discourse/lib/raw-render-glimmer";
+
+export default class extends EmberObject {
+  get html() {
+    return rawRenderGlimmer(
```

Comment:
We could wrap this with `if(connectorsExist(this.name))` to avoid paying for the RenderGlimmer (and wrapper element) when there are no connectors registered